deluge/ui/gtk3/piecesbar.py

```diff
@@ -26,15 +26,15 @@
 
 
 class PiecesBar(DrawingArea):
-    # Draw in response to an expose-event
-    __gsignals__ = {b'draw': b'override'}
+    # Draw in response to an draw
+    __gsignals__ = {'draw': 'override'}
```

Comment:
Can you explain why you reverted this bytes change? I was getting errors when using unicode strings.